src/storage/storage.ts

```diff
@@ -913,4 +914,32 @@ export class Storage implements IStorage {
     const accessData = await this._getValidAccessData();
     return (accessData.walletFlags & WALLET_FLAGS.HARDWARE) > 0;
   }
+
+  /**
+   * Return if the nano contract is registered for the given address based on ncKey.
+   * @param ncKey Pair address:ncId concatenated.
```

Comment:
Why do you need the pair and not only the `ncId`? You can have each `ncId` registered only once.

Reply:
Yeah! I will change it.